foo_ui_columns/status_bar.cpp

```diff
@@ -148,6 +149,12 @@ int calculate_selected_length_size(const char* p_text)
         win32_helpers::status_bar_get_text_width(g_status, "0d 00:00:00"));
 }
 
+int calculate_selected_count_size(const char* p_text)
+{
+    return std::max(win32_helpers::status_bar_get_text_width(g_status, p_text),
+        win32_helpers::status_bar_get_text_width(g_status, "00 tracks"));
```

Comment:
I think I'd bump this up in size a bit as it seems a bit inconsistent compared to the length of the selected tracks:

```suggestion
        win32_helpers::status_bar_get_text_width(g_status, "1,000 tracks"));
```

Reply:
Done